importers.md

```diff
@@ -121,89 +121,105 @@ Imagens enviadas por FTP,myfolder/image.jpg||myfolder/image2.jpg
-  * **Delimitador csv**: caractere que separa valores;
-  * **Delimitador de metadado multi-valorado**: caractere que separa valores dentro de uma mesma célula;
-  * **Delimitador de texto**: caractere que delimita todos os valores dentro de uma mesma célula
-  * **Codificação do arquivo**: parâmetro que determina a codificação dos valores de texto do aquivo. (Geralmente **UTF-8**, garanta que o arquivo *.csv* esteja codificado conforme as opções disponíveis no importador).
-6. Configure estes campos de acordo com suas preferências para importação:
-  * **Item repetido**: Determina o comportamento do Tainacan ao identificar itens idênticos no processo de importação. Selecione Atualizar para que o item receba os  valores do arquivo .csv ou seleciona Ignorar para que o item já existente na coleção não seja modificado.
-  * **Importando Anexos**: O Importador permite que vários itens sejam inseridos em uma coleção diretamente de um arquivo .csv. Consulte Importador-csv#Importando  arquivos e anexos para saber como configurar seu arquivo .csv corretamente.
-    1. De acordo com a documentação, aponte a URL no campo caminho para o servidor.
+### Import .csv to Tainacan
+
+1. Access the *WordPress* control panel
+2. In the left sidebar, click Tainacan
+3. Go to **Importers**
+4. In the **Available Importers** section, select `CSV`
+5. Configure the following fields according to the settings made in your *.csv* file:
+  * **Csv delimiter**: character that separates values within the same cell
+  * **Multi-value metadata delimiter**: character that separates values within a single cell
+  * **Text delimiter**: character delimiting all values within a single cell
+  * **File encoding**: parameter that determines the encoding of the text values in the file. (Usually **UTF-8**, make sure that the *.csv* file is encoded according to the options available in the importer).
+6. Configure these fields according to your import preferences:  
+  * **Item repeated**: Determines the behavior of Tainacan by identifying identical items in the import process. Choose Update so that the item receives the values from the *.csv* file, or choose ignore so that the item already in the collection is not changed.
+  * **Importing attachments**: The importer allows multiple items to be inserted into a collection directly from a *.csv* file. See [importing files and attachments](#import-files-and-attachments) to know how to configure your .csv file properly.
+    1. According to the documentation, point the URL in the path field to the server. 
+    
+    
+    
+    
+    
+    
 7. Selecione ou crie uma **Coleção Destino** para indicar onde os itens serão criados;
    *. Selecionando criar uma nova coleção em branco, ao `Concluir` a coleção, você será redirecionado para o **Importador** novamente.
+   
+   
+   
+   
+   
+8. Select or create a **Fate Collection** to indicate where the items will be created
```

Comment:
the meaning of "Coleção Destino" here is to indicate that this is the collection where the items will be imported to. I believe earlier you mentioned it as "Destination Collection", which seems to make more sense :)

Reply:
Ok it seems that this one was solved in the following commit, thanks :+1: